<launch>	
  <!-- Launch AR Glass Driver -->
  <node pkg="ar_glass" type="ar_glass_driver.py" name="ar_glass_driver" output="screen">
      <rosparam command="load" file="$(find ar_glass)/config/parameters.yaml" />
  </node>
</launch>
